frontend/device/pocketbook/device.lua

```diff
@@ -641,6 +641,24 @@ local PocketBook634 = PocketBook:extend{
     hasNaturalLight = yes,
 }
 
+-- PocketBook Verse Pro Color (PB634K3)
+local PocketBook634K3 = PocketBook:extend{
+    model = "PBVerseProColor",
+    display_dpi = 300,
+    hasColorScreen = yes,
+    canHWDither = yes, -- Adjust color saturation with inkview
+    canUseCBB = no, -- 24bpp
+    isAlwaysPortrait = yes,
+    hasNaturalLight = yes,
+    -- c.f., https://github.com/koreader/koreader/issues/9556
+    inkview_translates_buttons = true,
+}
+
+function PocketBook634K3._fb_init(fb, finfo, vinfo)
+    -- Pocketbook Color Lux reports bits_per_pixel = 8, but actually uses an RGB24 framebuffer
```

Comment:
Wrong device name in the comment ;). (Which, granted, I seem to remember is wrong across *all* the c/p instances of this comment ^^).

We might actually want to remove this comment in all subsequent copies, now that I think about it... (i.e., only keep the one in the first appearance of this hack).

Reply:
@ElimGarak1 Could you remove the comment since that's what @NiLuJe would like?